dataloader.go

```diff
@@ -219,6 +231,9 @@ func (l *Loader[K, V]) Load(originalContext context.Context, key K) Thunk[V] {
 		}
 		result.mu.RLock()
 		defer result.mu.RUnlock()
+		if result.value.Error != nil && reflect.TypeOf(result.value.Error) == reflect.TypeOf(&PanicErrorWrapper{}) {
```

Comment:
I don't like that you are using reflection here. Instead try to use the `Wrap` and `Is` functions. 
https://pkg.go.dev/errors#example-Is
https://pkg.go.dev/errors#Unwrap

Reply:
Yep! I'll make that change 👍